src/26-js-ipfs-0-24/index.md

```diff
@@ -0,0 +1,68 @@
+---
+baseurl: ..
+template: tmpl/layouts/post.html
+breadcrumbs:
+  - {name: "js-ipfs-0-24", link: "./" }
+id: 26-js-ipfs-0-24
+date: 2017-05-25
+title: js-ipfs 0.24.0 released
+author: David Dias
+collection: posts
+---
+
+I am pleased to announce to everyone in our Community that js-ipfs 0.24.0 has been successfully launched! This new minor release brings new features, bug fixes and new examples so that you can jump and start hacking your IPFS enabled apps right away!
+
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/IPFSbot">@IPFSbot</a> Woot! As promissed, new version of js-ipfs released!<br><br>🚀 `ipfs@0.24.0 [18:39] Published to npm.` 🚀<br><br>Release log here: <a href="https://t.co/20O2L2Scq5">https://t.co/20O2L2Scq5</a></p>&mdash; David Dias (@daviddias) <a href="https://twitter.com/daviddias/status/867512323732365312">May 24, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Highlights
+
+### 👢 WebSockets DNS Bootstrappers by default on the browser
+
+With this release, you won’t need to connect to Bootstrapper nodes manually, they will dialled from the start through their WebSockets endpoints. This also means that we’ve successfully deploy DNS support for multiaddrs, so that you can host your js-ipfs enabled app behind an HTTPS domain and dial to our Bootstrappers through WSS.
+
+### 🎈WebRTC is now a default transport
+
+We now include WebRTC multiaddr by default on new js-ipfs init calls. This means that both your Node.js and Browser nodes will be able to dial each other using WebRTC and discover other nodes in the network through Signalling Peer Discovery.
```

Comment:
is Signalling Peer Discovery a thing? Otherwise let's keep it lowercase. It would still be amazing if we could get some documentation on how webrtc-star works.